tools/modules/framework/manuals_markdown.sh

```
@@ -0,0 +1,221 @@
+	["markdown_manpage,feature"]="markdown_manpage"
+	["markdown_manpage,desc"]="Generate Document files."
+	["markdown_manpage,example"]=""
+	["markdown_manpage,group"]="Docs"
+)
+#
+# Function to generate the README.md file
+#
+function markdown_manpage() {
+
+	# Get the current date
+	local current_date=$(date)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)